packages/lib-classifier/src/components/Classifier/components/QuickTalk/components/Comment.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+import Comment from './Comment'
+
+const comment = {
+  id: '200001',
+  body: 'This is the first comment',
+  is_deleted: false,
+  user_id: '300001',
+  user_login: 'zootester',
+}
+
+const author = {
+  id: '300001',
+  avatar_src: null,
+  credited_name: 'zootester',
+  display_name: 'Zooniverse Tester',
+  login: 'zootester',
+}
+
+const authorRoles = [
+  { name: 'admin', section: 'project-1234' }
+]
+
+describe('Component > QuickTalk > Comment', function () {
+  beforeEach(function () {
+    render(
+      <ul>
+        <Comment
+          author={author}
+          comment={comment}
+          roles={authorRoles}
+        />
+      </ul>
+    )
+  })
+
+  it('should render the comment', function () {
+    expect(screen.getByText('This is the first comment')).to.exist()
+  })
+
+  it('should render the author details', function () {
+    expect(screen.getByText('Zooniverse Tester')).to.exist()
+    expect(screen.getByText(/\(@\s*zootester\s*\)/)).to.exist()
+
+    expect(screen.getByAltText('Avatar for Zooniverse Tester')).to.exist()
+    expect(screen.getByAltText('Avatar for Zooniverse Tester')).to.have.property('src', 'https://static.zooniverse.org/www.zooniverse.org/assets/simple-avatar.png')
```

Comment:
👍🏻

Not blocking, but should the alt text be localised?

Reply:
All alt text and Aria labels should be localised, _ideally._

Last night I was struggling with getting the [i18n English translations to be injected into the tested components,](https://react.i18next.com/misc/testing) as without that working, I'd always end up with literally `<button role="button" aria-label="QuickTalk.aria.openButton" />` instead of `<button role="button" aria-label="Click to open Quick Talk" />`

Alternative would be to just use (literally) `screen.getByAltText('QuickTalk.aria.AvatarAltText')` but I honestly can't tell if querying by translation key instead of translated text is better or worse. (e.g. RetiredBanner.spec.js checks for the string "Banners.AlreadySeenBanner.bannerText" instead of the actual English text)